dreambooth/train_dreambooth.py

```diff
@@ -1642,8 +1642,13 @@ def lora_save_function(weights, filename):
                         else:
```

Comment:
This already test for SDXL i think you can remove second test of SDXL after else and send values with dim=1 directly ? i will test on my environment because i got error generanting training for SDXL.

Reply:
im training now, lora on SDXL with classes, and its working.